src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -1,5 +1,17 @@
-(function(global, doc, eZ, $) {
+(function (global, doc, eZ, $) {
     const TOOLTIPS_SELECTOR = '[title]';
+    const config = {
+        attributes: true,
```

Comment:
Correct me if I'm wrong but I think we don't have to observe the `attributes`?

Reply:
yup, you have right